src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -218,12 +218,21 @@ const getColors = ({priority, disabled, borderless, theme}: StyledButtonProps) =
 
 const StyledButton = styled(
   React.forwardRef<any, ButtonProps>(
-    ({forwardRef, size: _size, external, to, href, ...props}: Props, forwardRefAlt) => {
+    (
+      {forwardRef, size: _size, external, to, href, ...otherProps}: Props,
+      forwardRefAlt
+    ) => {
       // XXX: There may be two forwarded refs here, one potentially passed from a
       // wrapped Tooltip, another from callers of Button.
 
       const ref = mergeRefs([forwardRef, forwardRefAlt]);
 
+      // only pass down title to child element if it is a string
```

Comment:
Hmm wait what why?

Reply:
Oh, this is for the title attribute I see

Reply:
✅

Reply:
yea not sure if this is the right way to do it, but because I changed the title prop for the button we couldn't pass it down to the child elements. Tbh unsure if we care that the children get `title`, but figured that conditionally including the prop would be safest.